src/test/java/com/iota/iri/service/tipselection/impl/WalkerAlphaTest.java

```diff
@@ -58,8 +58,7 @@ public static void setUp() throws Exception {
         tangle.init();
 
         TailFinder tailFinder = Mockito.mock(TailFinder.class);
-        Mockito.when(tailFinder.findTail(Mockito.any(Hash.class)))
-                .then(args -> Optional.of(args.getArgumentAt(0, Hash.class)));
+        Mockito.when(tailFinder.findTail(Mockito.any(Hash.class))).then(args -> Optional.of(args.getArgument(0)));
```

Comment:
I actually like it that the line was broken down. I found it more readable.
I need to play with the `format_settings.epf` file so it won't format the lines this way.

Reply:
OK, will change that.